### ISSUE TITLE
`iebaltab`: layout updates

### ISSUE THREAD

**Issue by luizaandrade** [worldbank/ietoolkit#335]
I have a project at the Lab that needs to create a balance table with two panels. So I figured I'd implement this in `iebaltab`. My idea was to have a similar workflow as in `estout` by adding the following options:
- `Fragment`
- `prehead` & `posthead` to customize headers (and thus solve #236)
- `prefoot` & `postfoot` to customize footers
- #147 
- #144 (if not yet implemented)

anything else @kbjarkefur and @bbdaniels think should be included here?

**Comment by kbjarkefur** [comment]
Would this only be for LaTeX? If not, then I think we should be specific what we want to do for LaTeX only and what we want to do for Excel and browse.

**Comment by kbjarkefur** [comment]
I have a strong preference to no add separate options that specifically suppress certain stats. Instead, I think this is intended to be a part of the `stats()` option, see this issue: https://github.com/worldbank/ietoolkit/issues/293. A big part of my implementation in the re-write was to move away with the ad-hoc approach of separate options for specific layout options.

So that `stats(desc(mean/se))` would be how you configure what is currently the default (happy to discuss whether that should be the default), and to drop the SE below mean you would do `stats(desc(mean))`.

**Comment by luizaandrade** [comment]
Yes, this would be only for LaTeX. I agree on the `stats()` option. I don't think the suggested option from #147 should be the default. If this is already implemented as `stats(desc(mean))`, I think we can mark that issue as solved.

**Comment by kbjarkefur** [comment]
I limited the scope of my big re-write to gather all already existing separate stats display options into `stats()`, but I did not add any new functionality to it. I had to limit the scope somehow to ever get that version out the door.

So instead of the `stdev` option you can now do `stats(desc(sd))`, but just as before the only option you have is to change what is displayed on the second row, not changing that there is no second row or that it is empty.

I think the solution is to allow users to set one or many stats in all of `stats(desc() pair() f() feq())` and then the tables adjust to have as many rows needed for each of them. I have more thinking on the syntax for setting multiple options, but we should probably discuss that in the issue for that option.

**Comment by bbdaniels** [comment]
I like this solution -- I should certainly be able to request `stats( [...] pair(beta f p))`!
